Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/InteractiveMeshCursor.cs

```diff
@@ -49,7 +49,7 @@ public class InteractiveMeshCursor : BaseCursor
         private bool hasHand = false;
         private bool isDown = false;
 
-        private Vector3 targetScale;
+        private new Vector3 targetScale;
```

Comment:
You marked targetScale as protected in BaseCursor.cs, is there a reason you decided to hide it (and make it private) in this class?

Reply:
I did it this way as I'm not really familiar with InteractiveMeshCursor (we don't use it), and so I wanted to just keep the behavior the same (while also wanting to minimize the amount of code changed). If I used the base version instead of hiding, I think it would potentially do a double-lerp of the scale (InteractiveMeshCursor.UpdateCursorTransform calls base.UpdateCursorTransform, and both do lerps, and not quite in the same way).
I'd be happy to rename the variable if you think that would be better, but I'd be hesitant to make any more substantial changes myself due to lack of familiarity.

Reply:
@davidkline-ms, any recommendations for @naostranMS here?

Reply:
I dont have any real recommendation, it just seemed odd to hide a base class' protected member with a derived class private member.

Reply:
> I think it would potentially do a double-lerp of the scale (InteractiveMeshCursor.UpdateCursorTransform calls base.UpdateCursorTransform, and both do lerps, and not quite in the same way).

Question: how does making targetScale private prevent this?

Reply:
It being private doesn't really matter to this change. I meant that I could have just removed this targetScale and used the now accessible (protected) base class targetScale instead, but then I would be worried about it doing two lerps.
Would you prefer if I changed this to be protected also?

Reply:
Looking at the code, it appears to do two lerps regardless of private or protected :)

I, personally, am not a fan of hiding varaiables.

Thx!

Reply:
fixed!